src/filetypes.c

```diff
@@ -951,7 +951,7 @@ static void load_settings(guint ft_id, GKeyFile *config, GKeyFile *configh)
 	{
 		ft->lang = tm_source_file_get_named_lang(result);
 		if (ft->lang < 0)
-			geany_debug("Cannot find tag parser '%s' for custom filetype '%s'.", result, ft->name);
+			geany_debug("Cannot find tags parser '%s' for custom filetype '%s'.", result, ft->name);
```

Comment:
For me "symbols parser" is something which parses symbols, not something whose output are symbols (like "C parser" is something which parses C files). The "tags parser" feels more like "ctags parser" so not something which parses tags files but rather "a parser that is part of ctags".

Also notice in which case this message is written by looking a few lines above in the code - it's written for the "tag_parser" settings item and if we don't want to rename it, I'd rather keep the "tags parser" naming.